test/journal_test.go

```diff
@@ -111,14 +111,19 @@ func TestJournalCrSimple(t *testing.T) {
 	)
 }
 
-// bob creates many conflicting files while running the journal.
-func TestJournalCrManyFiles(t *testing.T) {
-	var busyWork []fileOp
-	iters := 20
+func giveMeHiWork(iters int) (busyWork []fileOp) {
```

Comment:
If the caller needs to know the fliename, maybe just have them pass it in?  It seems awkward in the function name.